client/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<link rel="stylesheet" type="text/css" href="css/self-check.css" />
+	<link href="https://fonts.googleapis.com/css?family=Ubuntu:300|Orienta" rel="stylesheet" type="text/css">
+	<script type="text/javascript" src="js/self-check.js"></script>
+	<title>Buswell Library - Self Check</title>
+</head>
+
+<body onload="javascript:initiate();">
+	<div class="toptoolbar">
+		<div class="logo">
+			<img src="images/logo.png" height="52px" alt="Wheaton College Buswell Library Logo" />
+		</div>
+	</div>
+	<br />
+
+	<div class="maincontent smallbox" id="loginbox">
+		<div class="h1title">Self-Check Service</div>
+		<br /><br />
+		<div class="databox">
+			<div class="h2title" id="idinstruction">Please scan or type your ID:</div>
+			<div>
+				<div class="hide errormessage" id="loginerror">Incorrect ID, please try again<br /></div>
+				<input type="text" id="userid" placeholder="ID..." />
+				<br /><br />
+				<input type="button" id="submit" value="Submit" onclick="javascript:login();" />
+			</div>
+		</div>
+		<br /><br />
+	</div>
+
+	<div class="maincontent largebox hide" id="scanbox">
+		<div class="h1title" id="scanboxtitle"></div>
+		<div class="databox">
+			<div>
+				<table cellpadding="0px" cellspacing="0px" class="headertable">
+					<tr>
+						<td><span class="headertablekey">Loans:</span>&nbsp;<span class="headertablevalue"
+								id="userloans"></span></td>
+						<td><span class="headertablekey">Requests:</span>&nbsp;<span class="headertablevalue"
+								id="userrequests"></span></td>
+						<td><span class="headertablekey">Fines & Fees:</span>&nbsp;<span class="headertablevalue"
+								id="userfees"></span></td>
+						<!-- <td><span class="headertablekey">Notes:</span>&nbsp;<span class="headertablevalue" id="usernotes"></span></td> -->
+						<td><span class="headertablekey">Timeout: </span><span class="headertablevalue"
+								id="usertimeout"></span></td>
+						<td id="logoutinstruction">Scan your ID again to logout or <input type="button" value="Logout"
+								onclick="javascript:logout();" /></td>
+
+					</tr>
+				</table>
+			</div>
+			<br /><br />
+			<div class="h2title" id="iteminstruction">Please scan or type your next item:</div>
+			<div>
+				<input type="text" id="barcode" placeholder="Barcode..." />
+			</div>
+			<br />
+			<div>
+				<table cellpadding="0px" cellspacing="0px" class="loanstable" id="loanstable">
+					<thead>
+						<tr>
+							<th>Title</th>
+							<th>Due Date</th>
+						</tr>
+					</thead>
+					<tbody>
+					</tbody>
+				</table>
+			</div>
+		</div>
+		<br /><br />
+	</div>
+	<br /><br />
+
+	<!-- Trigger/Open The Modal -->
+	<div id="myModal" class="modal hide">
+
+		<!-- Modal content -->
+		<div class="modal-content" id="modalbox">
+			<span id="modalclosex">x</span>
```

Comment:
This is the modal that has the id set to `modalclosex`, but no class is set, even though the CSS selectors targeting this expect there to be `class="modalclosex"`. Either change the CSS selector to target this ID or add the necessary class here.

Reply:
4d00a20d fixed this.